chia/wallet/puzzles/dao_proposal.clvm

```diff
@@ -45,6 +46,7 @@
   (defconstant dao_finished_state 0xfb015415f2e6a09c1141f880fc2135beec6adf2a19e4d02a191432846db16559)
 
   (defun-inline calculate_win_percentage (CURRENT_CAT_ISSUANCE PROPOSAL_PASS_PERCENTAGE)
+    ; COMMENT: this code would be clearer if the divmod was performed on the proposal_pass_percente/ten_thousand first, then multiplied to current_cat_issuance (maybe more optimal too not sure)
```

Comment:
with our division flooring, I think it's more accurate and safer to calculate the big number first